### PR TITLE
Bugfix/dropdown menu trigger as child remove class names

### PR DESCRIPTION
**Pull request by emrysal**
## What does this PR do?

* Fixes to Dropdown styling
* Fixes the CreateEventTypeButton mobile version
* Fixes UsernameTextfield input height
* Cleaned yarn.lock slightly

### CODE REVIEW

File: apps/web/components/ui/UsernameAvailability/UsernameTextfield.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ const UsernameTextfield = (props: ICustomUsernameProps) => {
             autoCapitalize="none"
             autoCorrect="none"
             className={classNames(
-              "mb-0 mt-0 h-6 rounded-md ltr:rounded-l-none rtl:rounded-r-none",
+              "mb-0 mt-0 rounded-md ltr:rounded-l-none rtl:rounded-r-none",
```
Comment (emrysal) [inline apps/web/components/ui/UsernameAvailability/UsernameTextfield.tsx:134]: Unlocks the height of the username input - fixes incorrect height of the input field.

File: packages/features/ee/teams/components/MemberListItem.tsx [hunk 2]
```diff
@@ -149,13 +149,11 @@ export default function MemberListItem(props: Props) {
               </Tooltip>
               {editMode && (
                 <Dropdown>
-                  <DropdownMenuTrigger
-                    asChild
-                    className="h-[36px] w-[36px] bg-transparent px-0 py-0 hover:bg-transparent focus:bg-transparent focus:outline-none focus:ring-0 focus:ring-offset-0">
```
Comment (emrysal) [inline packages/features/ee/teams/components/MemberListItem.tsx:154]: Most of this is handled by the Button component except the radix-state-open not present here; added to the Button component and removed all classNames.

File: packages/features/eventtypes/components/CreateEventTypeButton.tsx [hunk 3]
```diff
@@ -331,80 +326,3 @@ export default function CreateEventTypeButton(props: CreateEventTypeBtnProps) {
     </>
   );
 }
-
-type CreateEventTypeTrigger = {
```
Comment (emrysal) [inline packages/features/eventtypes/components/CreateEventTypeButton.tsx:335]: Dead code.

File: packages/ui/components/form/dropdown/Dropdown.tsx [hunk 4]
```diff
@@ -14,11 +14,10 @@ export const DropdownMenuTrigger = forwardRef<HTMLButtonElement, DropdownMenuTri
   ({ className = "", ...props }, forwardedRef) => (
     <DropdownMenuPrimitive.Trigger
       {...props}
-      className={
-        props.asChild
-          ? classNames("rounded-md ring-0", className)
```
Comment (emrysal) [inline packages/ui/components/form/dropdown/Dropdown.tsx:19]: Broke mobile styling of the CreateEventTypeButton.

**Review by zomars — APPROVED** [review]

**Pull request merged**